yamllint/rules/quoted_strings.py

```diff
@@ -112,6 +115,28 @@
 
     - "localhost"
     - this is a string that needs to be QUOTED
+
+#. With ``quoted-strings: {quote-type: double, required: true,
+   skip-quoted-quote: false}``
+
+   the following code snippet would **PASS**:
+   ::
+
+    foo: "bar\"baz"
+
+   the following code snippet would **FAIL**:
+   ::
+
+    foo: 'bar"baz'
+
+#. With ``quoted-strings: {quote-type: double, required: true,
+   skip-quoted-quote: true}``
```

Comment:
```suggestion
#. With ``quoted-strings: {skip-quoted-quote: true}``
```

Reply:
I've dropped `required: true` and kept `quote-type: double` because without `quote-type: double` yamllint won't complain about quotes at all.